### DISCUSSION TITLE
Update The Offline Weather Icon With The News and Interests Offline Icon

### DISCUSSION THREAD

**Discussion by milestprower92** [valinet/ExplorerPatcher#2263] [Ideas]
There are the offline news and interests Icon Light And Dark Mode Versions
![image](https://github.com/valinet/ExplorerPatcher/assets/131640765/c89c67de-bc3b-4bb7-a3fa-8055245f4600)
![image](https://github.com/valinet/ExplorerPatcher/assets/131640765/fa923f43-72f3-4827-b7c4-a7b01623004c)

**Comment by XuanChen1234** [comment]
![屏幕截图 2023-12-12 202938](https://github.com/valinet/ExplorerPatcher/assets/140480110/8df3eb41-5da6-466a-a196-f8a9aefff762)
